### ISSUE TITLE
[Feature Request] Remove items

### ISSUE THREAD

**Issue by Gandalf-the-Blue** [IgnisDa/ryot#57]
Basically title, add an easy way to remove items from "My Books","My Shows" etc.

**Comment by IgnisDa** [comment]
So a media item will come under "My Books" only if sone conditions match (here https://github.com/IgnisDa/ryot/blob/124b5d9cda663f5103d5bfd4984122c4f56956f7/apps/backend/src/migrator/m20230417_000004_create_user.rs#L10-L14). If not, they will be removed automatically by a cron job.

**Comment by Gandalf-the-Blue** [comment]
Yes, I believe I jumped the gun here. The cron job took care of that item.
